templates/validator/dynconf_update.sh.epp

```diff
@@ -51,3 +53,18 @@ for data in blacklists ticklemacros issuer_keyring participants products service
 done
 
 test -x /usr/local/bin/kdk_update && /usr/local/bin/kdk_update
+
+<%if $bobcat::validator::refresh_api { -%>
+
+if [ $? -eq 0 ]; then
+        REFRESH=true
+fi
+
+if [ "$REFRESH" = true ] ; then
+	#let files cool down
```

Comment:
Are they hot? Once they are written by curl they will not go anywhere.

Reply:
true ill change this